doc/configuration.rst

```diff
@@ -445,6 +445,23 @@ Currently only bzr and git repositories get reverted.
 
 .. note:: new in version 1.9.0
 
+vcs-clear-retry
+---------------
+
+If ``True`` failed updates are cleared and retried once.
+This is intended for brittle VCSes from CI robots.
+
+.. note:: new in version 1.9.0
+
+vcs-clear-locks
+---------------
+
+Some VCS systems can leave locks after some failures and provide a separate
+way to break them. If ``True``,the repo will break any locks prior to
+operations (mostly useful for automated agents, such as CI robots)
+
+.. note:: new in version 1.9.0
```

Comment:
Thanks, I would never had imagined they weren't documented.
These are really not new, but at least 2 years old (I'll take care of that)
There is a new related behaviour in the Git subsystem (a reset --hard is tried as a faster solution before defaulting to the clear-retry)